src/utils/general-helpers.ts

```diff
@@ -74,3 +74,6 @@ export const getCurrentPath = () => {
     return pathname;
   }
 };
+
+export const getObjectEntriesWithValue = (object: Record<string, any>) =>
```

Comment:
should it be called recordEntries since its not for all objects?

Reply:
Er ikke object egentlig bare en Record? 🤔 Ser ut som det virker likt om jeg endrer `Record<string, any>` til `object` så tenker det da er greit nok